src/subscription_manager/managerlib.py

```diff
@@ -954,7 +954,8 @@ def allows_multi_entitlement(pool):
     top-level product's multi-entitlement attribute.
     """
     for attribute in pool['productAttributes']:
-        if attribute['name'] == "multi-entitlement" and \
-            utils.is_true_value(attribute['value']):
+        if attribute['name'] == "multi-entitlement" and utils.is_true_value(
+            attribute['value']
+        ):
```

Comment:
When I look at it now, I agree. There's also a option of
```py
if (
    attribute['name'] == "multi-entitlement"
    and utils.is_true_value(attribute['value'])
):
```
which I personally prefer over indentation of the `and` in your suggestion, because then the block under the condition is visually separated form the condition itself.